spec/features/log_out_inactive_admins_spec.rb

```diff
@@ -25,7 +25,6 @@
         Timecop.travel(login_time + 29.minutes)
         visit admin_feature_url
 
-        expect(page).to have_http_status(:success)
```

Comment:
pages were getting 304 Not Modified, but not getting sent to login (which this spec is worried about), so removed this check and the one below